### PR TITLE
Light Mode: extract fixture module, add cost models

### PR DESCRIPTION
**Pull request by Unisay**
### Issue Number

ADP-1572

Cost models are taken from the corresponding Alonso genesis configuration files:

1. [Mainnet](https://github.com/input-output-hk/cardano-node/blob/0f6a20d15b69f599eafc3d46535474ba67c52693/configuration/cardano/mainnet-alonzo-genesis.json#L26)
2. [Testnet](https://github.com/input-output-hk/cardano-node/blob/0f6a20d15b69f599eafc3d46535474ba67c52693/configuration/cardano/testnet-alonzo-genesis.json#L26)

### CODE REVIEW

File: lib/shelley/src/Cardano/Wallet/Shelley/Network/Blockfrost.hs [hunk 1]
```diff
@@ -786,155 +774,155 @@ instance FromBlockfrost BF.Block BlockHeader where
...
+                    , protocolParamMonetaryExpansion = toRational _protocolParamsRho
+                    , protocolParamTreasuryCut = toRational _protocolParamsTau
+                    , protocolParamUTxOCostPerWord =
+                        Just $
+                            Node.Lovelace $
+                                intCast _protocolParamsCoinsPerUtxoWord
+                    , protocolParamCostModels =
+                        Map.singleton
+                            (AnyPlutusScriptVersion PlutusScriptV1)
+                            (Fixture.costModels network)
```
Comment (HeinrichApfelmus) [inline lib/shelley/src/Cardano/Wallet/Shelley/Network/Blockfrost.hs:899]: 👍

For the hard-fork, we'll need to add a cost model for `PlutusScriptV2` as well. Not sure if it's publicly available yet. Could you make a ticket so we do not forget?

**Review by HeinrichApfelmus — APPROVED** [review]
Looks good to me! 😊

File: lib/shelley/src/Cardano/Wallet/Shelley/Network/Blockfrost.hs (hunk 1 above)
Comment (Unisay) [reply lib/shelley/src/Cardano/Wallet/Shelley/Network/Blockfrost.hs:899]: https://input-output.atlassian.net/browse/ADP-1804

**Pull request merged**